controller_definition.py

```diff
@@ -436,23 +436,33 @@ def adjustMixer(self, knob: int, dataType: str, action: str, selectedTrack: int)
         # Multiplies the trackGroup to 8 to get the index of the first track of that group
         trackFirst = trackGroup * 8
 
+        # If you receive a sensitivity value, set it to a controllable value, else default to 1
+        if (sensitivity):
+            sensitivity = sensitivity / 15
+
+            # When decreasing the mixer knob, the amount that it changes is dramatically different than increasing.
+            if (action == "DECREASE"):
+                sensitivity = sensitivity / 10
+        else:
+            sensitivity = 1
+
```

Comment:
No need to do any of this to normalize the value. Try first with the suggestion I wrote in my review and tweak the logarithmic function before using falling back the implementation a conditional block again.